backend/data_distro/api_tests/schema/passthrough.jsonnet

```diff
@@ -0,0 +1,15 @@
+local Base = import 'APITestBase.libsonnet';
+local Types = Base.Types;
+
+Types.object {
+  properties: {
+    id: Types.integer,
+    passthrough_name: Types.string,
+    passthrough_id: Types.StringOrNull, // This doesn't seem like something that should be optional, but there are nulls in the data 
```

Comment:
There are a few cases like this, where it doesn't seem like `null` should be possible, but there are multiple `null`s in the data.

Reply:
I think, with this (and the question about custom types above) that we might ticket out separately. That is, the schema refactor/cleanup team have a number of things that can/should be refined/improved. However, *as a first increment*, getting all the schemas in one place, and... *enbettered*... is good.

I would say before closing out the ticket, we should either 1) start a new one, or 2) tag on to a ticket with the other schema refactor team, and 3) add checklist items for exactly these kinds of questions. E.g. "Revisit all fields that contain null, and decide whether `null` should be allowed" or similar. It pushes the question down the road a bit, but it also lifts it up into view, and forces us to address it.

In other words, I think these schemas do more good *in the tree* and ready to use *in an incremental state* than halting forward progress to solve/answer these questions.